src/hosting/normalizedHostingConfigs.ts

```diff
@@ -56,6 +56,41 @@ function filterOnly(configs: HostingConfig[], onlyString: string): HostingConfig
   return filteredConfigs;
 }
 
+function filterExcept(configs: HostingConfig[], exceptString: string): HostingConfig[] {
```

Comment:
Nit: `exceptString` seems like a redundant name since we already declare its type. Prefer just `except` instead.

Reply:
In context, it's not important that it's a "string" by the name, but that it's the "option string" from the CLI options object. I'll go with `exceptOption: string` to make it clear (it's used quickly and briefly, so that should work).